support-frontend/assets/pages/supporter-plus-landing/components/threeTierCards.tsx

```diff
@@ -39,13 +40,18 @@ export function ThreeTierCards({
 	paymentFrequency,
 	cardsCtaClickHandler,
 }: ThreeTierCardsProps): JSX.Element {
+	const haveRecommendedAndSelectedCards =
+		cardsContent.filter((card) => card.isRecommended || card.isUserSelected)
+			.length > 1;
+
 	return (
 		<div css={container(cardsContent.length)}>
 			{cardsContent.map((cardContent, cardIndex) => {
 				return (
 					<ThreeTierCard
 						key={`threeTierCard${cardIndex}`}
 						{...cardContent}
+						isRecommendedSubdued={haveRecommendedAndSelectedCards}
```

Comment:
Should we be passing `isUserSelected` in here too?

Reply:
it gets passed through as part of the spread on the line above